src/commands/config/index.ts

```diff
@@ -114,12 +147,20 @@ class ConfigCommand extends Command {
         response = await karmaTrackingSubcommand(interaction);
         break;
       }
+      case ConfigSubcommandGroup.Logs: {
+        response = await logsSubcommand(interaction);
+        break;
+      }
       case ConfigSubcommandGroup.RandomVoiceChannelNames: {
         response = await randomVoiceChannelNamesSubcommand(interaction);
         break;
       }
     }
 
+    if (!response) {
+      response = 'Something went VERY wrong';
```

Comment:
Maybe throw an error instead?

Reply:
Does something catch the errors that are thrown here? Or do they surface to the user?